components/OtherUserSpectrum.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <li>
+    <slot></slot>
+    <span class="other-user-spectrum-gradient" :class="gradientStyle">
+      <span class="other-user-circle" :style="{ left: positionStyle }"></span>
+    </span>
+  </li>
+</template>
+
+<script>
+export default {
+  props: {
+    /* all gradientStyle classes passed to this component are generated in ~/assets/scss/gradients.scss */
```

Comment:
I guess "generated" mean "defined"?

Reply:
There's a sass mixin generating the classes in the file, but defined is also good